src/app/pages/news/11.2/02-portal.md

```diff
@@ -1,10 +1,34 @@
 ## Axon Ivy Portal {#portal}
 
-Blah Portal blah.
+Introducing Axon Ivy Portal {#portal}! Packed with new features like notifications, enhancements to the Portal Dashboards, and multiple improvements for a smoother and more efficient user experience. 
+
+- __Notifications__:
+The portal seamlessly integrates the redesined notifications provider of the Axon Ivy engine. These notifications are prominently displayed within the portal interface, offering users the ability to filter them, mark notifications as read, and efficiently manage their preferences. Users can easily subscribe or unsubscribe to various notification channels, ensuring a tailored and personalized notification experience.
+
+- __Lazy Loading Task and Case Lists__:
+We have optimized the task and case lists by removing pagination in favor of implementing lazy loading.
+
+- __AI Translations__:
+With the possibility of integrating DeepL into the Portal, creating multi-language dashboards has become remarkably convenient. Widget titles and published news within the news widget can be translated effortlessly with the assistance of AI technology.
+
+- __Import, Export and Sharing of Portal Dashboards__:
+With Import/Export functionality, seamlessly move and duplicate dashboards across different enviroments. Sharing Dashboards empowers collaboration by allowing the sharing of customized dashboards via link with team members.
+
+- __Enhanced Accessibility for Custom Dashboard Widgets__:
+The custom widget can display a predefined Ivy process directly in the dashboard, embeded into a dashboard widget. If you use this functionality, selecting and adding available custom widgets to your dashboard has become much more straightforward!
+
+And much more: 
+
+- Custom Order of the Processes in the Dashboard Process Widget
+
+- More Enduser like Business States for Tasks and Cases
```

Comment:
Enduser -> end-users

Reply:
Important that you put here "End-user-like otherwise, the sentence has another meaning